### PR TITLE
upd bot and other

### PR DESCRIPTION
**Pull request by SpaceIgor**

### CODE REVIEW

**Review by OlegBondarUA — APPROVED** [review]

**Comment by SpaceIgor** [comment]
@WARGDOOM

**Comment by SpaceIgor** [comment]
@WARGDOOM

File: auto_bot/dispatcher.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def setup_dispatcher(dp):
     # incomplete auth
     dp.add_handler(MessageHandler(Filters.contact, update_phone_number))
     # ordering taxi
-    dp.add_handler(CallbackQueryHandler(start_query, pattern="Back_to_main"))
+    dp.add_handler(CallbackQueryHandler(start_query, pattern="Back_to_main|Main_"))
```
Comment (KurochkaR) [inline auto_bot/dispatcher.py:112]: Why add new callback instead using first one?

File: auto_bot/handlers/driver_manager/keyboards.py [hunk 2]
```diff
@@ -40,55 +41,62 @@ def inline_statistic_kb():
...
     keyboard = [
         [InlineKeyboardButton(f"{vehicle}", callback_data=f"{callback} {vehicle.id}")] for vehicle in vehicles]
-    keyboard.append([InlineKeyboardButton(order_inline_buttons[6], callback_data="Back_to_main")])
+    keyboard.append([InlineKeyboardButton(order_inline_buttons[6], callback_data=back_step)])
+    keyboard.append(main)
     return InlineKeyboardMarkup(keyboard)
 
 
-def inline_partner_drivers(callback, drivers, pk_vehicle=None):
+def inline_partner_drivers(callback, drivers, back_step, pk_vehicle=None,):
```
Comment (KurochkaR) [inline auto_bot/handlers/driver_manager/keyboards.py:63]: rm comma at the end

File: auto_bot/handlers/main/keyboards.py [hunk 3]
```diff
@@ -105,14 +106,12 @@ def inline_work_driver_kb():
...
-                                                                           created_at__date=timezone.now().date(),
-                                                                           end_at=None) else inline_work_driver_kb(),
-        "CLIENT": inline_user_kb(),
+        "DRIVER": inline_start_driver_kb(),
+        "CLIENT": inline_user_kb(ParkSettings.get_value('SHIPPING_CHILDS')),
         "DRIVER_MANAGER": inline_manager_kb(),
         "OWNER": inline_owner_kb()
     }
-    reply_markup = role_reply_markup.get(user.role, inline_user_kb())
+    reply_markup = role_reply_markup.get(user.role, inline_user_kb(ParkSettings.get_value('SHIPPING_CHILDS')))
```
Comment (KurochkaR) [inline auto_bot/handlers/main/keyboards.py:114]: why use args instead add url in keyboard?

File: auto_bot/handlers/main/keyboards.py [hunk 4]
```diff
@@ -132,3 +131,11 @@ def markup_keyboard(keyboard):
...
 def markup_keyboard_onetime(keyboard):
     return ReplyKeyboardMarkup(keyboard, resize_keyboard=True, one_time_keyboard=True)
+
+
+main = [InlineKeyboardButton(main_buttons[8], callback_data="Main_")]
+
+
+def back_to_main_menu():
+    keyboard = [main]
+    return InlineKeyboardMarkup(keyboard)
```
Comment (KurochkaR) [inline auto_bot/handlers/main/keyboards.py:141]: variable for what?

File: auto_bot/handlers/driver_manager/keyboards.py (hunk 2 above)
Comment (SpaceIgor) [reply auto_bot/handlers/driver_manager/keyboards.py:63]: +

File: auto_bot/handlers/main/keyboards.py (hunk 4 above)
Comment (SpaceIgor) [reply auto_bot/handlers/main/keyboards.py:141]: +

**Review by KurochkaR — APPROVED** [review]

**Pull request merged**